src/server/scripts/Shadowlands/SepulcherOfTheFirstOnes/boss_anduin_wrynn.cpp

```diff
@@ -0,0 +1,4307 @@
+/*
+ * This file is part of the TrinityCore Project. See AUTHORS file for Copyright information
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation; either version 2 of the License, or (at your
+ * option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "AreaTrigger.h"
+#include "AreaTriggerAI.h"
+#include "DB2Stores.h"
+#include "CellImpl.h"
+#include "TaskScheduler.h"
+#include "CreatureAI.h"
+#include "CreatureAIImpl.h"
+#include "GridNotifiers.h"
+#include "GridNotifiersImpl.h"
+#include "G3DPosition.hpp"
+#include "MoveSplineInitArgs.h"
+#include <G3D/Box.h>
+#include <G3D/CoordinateFrame.h>
+#include "InstanceScript.h"
+#include "Map.h"
+#include "MotionMaster.h"
+#include "MovementTypedefs.h"
+#include "Object.h"
+#include "ObjectAccessor.h"
+#include "PathGenerator.h"
+#include "Player.h"
+#include "ScriptMgr.h"
+#include "ScriptedCreature.h"
+#include "SpellAuras.h"
+#include "SpellScript.h"
+#include "SpellAuraEffects.h"
+#include "TemporarySummon.h"
+#include "UpdateFields.h"
+#include "G3D/Vector2.h"
+#include "Unit.h"
+#include "Weather.h"
+#include "GameObjectAI.h"
+#include "Containers.h"
+#include "SpellMgr.h"
+#include "ScriptedGossip.h"
+#include "sepulcher_of_the_first_ones.h"
+
+enum AnduinWrynnSpells
+{
+    // Pre-Introduction
+    SPELL_BROKER_SPAWN                              = 367524,
+
+    // Generic Spells
+    SPELL_ANDUIN_PLUNGE_KINGSMOURNE                 = 369125,
+    SPELL_WILLPOWER_DISPLAY                         = 365177,
```

Comment:
We should rename it to SPELL_POWER_DISPLAY_WILLPOWER, we've using lately  SPELL_POWER_DISPLAY_NAME.